TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -1100,7 +1100,7 @@ static object GetEnd(long start, long length)
         private List<Tuple<int, DataDefinition, int>> CollectDataLayoutNodes(CompilationUnit compilationUnit)
         {
             var dataLayoutNodes = new List<Tuple<int, DataDefinition, int>>();
-            Node dataDivision = compilationUnit.TemporaryProgramClassDocumentSnapshot.Root.MainProgram.GetChildren<DataDivision>().FirstOrDefault();
+            Node dataDivision = compilationUnit?.TemporaryProgramClassDocumentSnapshot?.Root?.MainProgram?.GetChildren<DataDivision>()?.FirstOrDefault();
```

Comment:
`compilationUnit`, `Root` and the result of `GetChildren<>` cannot be `null`. `TemporaryProgramClassDocumentSnapshot` should not be null either as we force node rebuilding. 

I'm ok to keep it like this safe approach but this will be changed if we move forward with the `#nullable` contexts.